test/v8-to-istanbul.js

```diff
@@ -89,6 +90,49 @@ ${'//'}${'#'} sourceMappingURL=data:application/json;base64,${base64Sourcemap}
       // that means it was bale to access the content via the provided sources object
       v8ToIstanbul.sourceTranspiled.should.not.be.undefined()
     })
+
+    it('should clamp line source column >= 0', async () => {
+      const v8ToIstanbul = new V8ToIstanbul(
+        `file://${require.resolve('./fixtures/scripts/needs-compile.compiled.js')}`,
+        0
+      )
+
+      // read the file and find the first end of line char
+      const fileBody = readFileSync(require.resolve('./fixtures/scripts/needs-compile.compiled.js')).toString()
+      const matchedNewLineChar = fileBody.match(/(?<=\r?\n)/u).index
+      assert(fileBody.substring(matchedNewLineChar - 2, matchedNewLineChar) === '\r\n', 'The test fixture is misconfigured!')
+      await v8ToIstanbul.load()
+      // apply a fake range that starts with the matched new line
+      // (these ranges can occur on v8 running on windows) and verify
+      // coverage is applied correctly. CovLine will have a
+      // gap between the endCol of the previous line ending on \r and startCol of the
+      // next line. This would cause -1 to be sent for the source map lookup.
+      // This would cause source map translation to throw
+      v8ToIstanbul.applyCoverage([{
+        functionName: 'fake',
+        ranges: [{
+          startOffset: matchedNewLineChar - 1,
+          endOffset: matchedNewLineChar + 10
```

Comment:
Why 10?

Reply:
No real reason. I just needed a range. Only the first char position is important for the bug to manifest.